src/main/docs/guide/dbc/dbcRepositories.adoc

```diff
@@ -34,4 +34,20 @@ As you can see from the above example, using abstract classes can be useful as i
 
 The example above uses the api:data.jdbc.runtime.JdbcOperations[] interface which simplifies executing JDBC queries within the context of transactions.
 
-TIP: You could also inject whichever other tool you wish to use to handle more complex  queries, such as QueryDSL, JOOQ, Spring JdbcTemplate etc.
+====
+
+You can also integrate whichever other tool you wish to use to handle more complex queries, such as QueryDSL, JOOQ, Spring JdbcTemplate etc.
+
+For example, to use link:https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/jdbc/core/JdbcTemplate.html[Spring JdbcTemplate],
+
+Add the following dependencies:
+
+dependency:micronaut-data-jdbc[groupId="io.micronaut.data"]
+dependency:micronaut-data-spring-jdbc[groupId="io.micronaut.data"]
```

Comment:
```suggestion
dependency:micronaut-data-jdbc[groupId="io.micronaut.data"]

dependency:micronaut-data-spring-jdbc[groupId="io.micronaut.data"]
```

Without a blank line, these two will smush into each other and look wrong I think